### ISSUE TITLE
Missing 'attr' config field in TextType doc

### ISSUE THREAD

**Issue by QuentinCurtet** [symfony/symfony-docs#9949]
Hello,

This doc : https://symfony.com/doc/current/reference/forms/types/text.html said that it's actually not possible to use "attr" option on a TextType but when we use it, it's ok and the 'attr' is affected to the field.

Example of config : 
```
'id' => 'MyId',
'type' => TextType::class?
'options' => [
	'label' => 'MyLabel',
	'label_attr' => ['class' => 'control-label'],
	'attr' => ['class' => 'form-control']
]
```

**Comment by HeahDude** [comment]
Hello @QuentinCurtet, thank you for opening this issue. The list of options if actually missing some inherited ones yes.
By checking all FormType options, you can see all of them as usable unless they appear in the (up to date) overridden section.

**Comment by QuentinCurtet** [comment]
Ok, thanks for your return !

**Comment by javiereguiluz** [comment]
Fixed by #10094.
